src/huggle_ui/blockuserform.cpp

```diff
@@ -133,6 +133,7 @@ void BlockUserForm::Block()
     this->timer->stop();
     if (this->ui->cbMessageTarget->isChecked())
         this->sendBlockNotice(nullptr);
+    this->close();
```

Comment:
line 125 is useless unless we implement that timer thing I mentioned in previous comment and the translation probably as well.

Reply:
added the timer here as well.